lib/pbench/test/unit/server/test_cache_manager.py

```diff
@@ -1689,8 +1719,11 @@ def test_compatibility(
         t2 = cm1[id]
         assert t1.name == t2.name == Dataset.stem(source_tarball)
 
-        t1.unpack()
-        t2.unpack()
+        r1 = t1.get_results(FakeLockRef(t1.lock))
+        assert r1 == t1.unpacked
+        r2 = t2.get_results(FakeLockRef(t2.lock))
+        assert r2 == t2.unpacked
+        FakeLockRef.reset()
```

Comment:
Since you were only calling `reset()` once in this test, a fixture would have worked fine, without breaking this case into two; and, in this particular case, there is, strictly speaking, no need for the second reset, because the test is not currently looking at the class attributes...but, it is better to have it in there so that any future extensions to this code aren't "surprised" by the state lingering from calls to the first mock.